requirements.txt

```diff
@@ -11,7 +11,7 @@ pymodbus==1.2.0
 pyserial==3.1.1
 python-dateutil==2.5.3
 pytz==2016.6.1
-pyzmq==15.4.0
```

Comment:
I assume there was a need to upgrade this?

Reply:
Yes, Jupyter now requires a fairly current version of pyzmq. We had a quick conversation about it a month or so ago. You thought that VOLTTRON's restriction to an older version of pyzmq was probably for reasons that are now obsolete.